tensor2tensor/data_generators/problem.py

```diff
@@ -931,6 +931,15 @@ def prepare_for_output(example):
 
     return dataset
 
+  @property
+  def export_assets(self):
```

Comment:
Hi @gcampax - I still see the old name here, have you done another commit after taking @lukaszkaiser 's review into account?

Thanks

Reply:
My bad, I misread his comments, I'll start the process of merging this in shortly!